Automattic/wp-calypso
# Domains: Use breadcrumbs on placeholder header

#### Changes proposed in this Pull Request
Adds the breadcrumbs component to the main domain placeholder component. This is part of the domain pages redesign project detailed in pcYYhz-m2-p2.

#### Mobile
##### Before
![image](https://user-images.githubusercontent.com/18705930/140438802-9e0e9926-8e5c-46a6-b7dd-8e9007b6f863.png)

##### After
![image](https://user-images.githubusercontent.com/18705930/140438317-4c00a655-7a23-4299-9fa5-e20c0d569e86.png)

---

#### Desktop
##### Before
![image](https://user-images.githubusercontent.com/18705930/140438811-298aa755-21e0-4cda-a3cc-fea38d0d876d.png)

##### After
![image](https://user-images.githubusercontent.com/18705930/140438687-27fe13ba-3a33-4c50-9900-cc7c293e0881.png)

---

#### Testing instructions
- Go to the domain name servers settings page (`/domains/manage/<site>/name-servers/<domain>`);
- Click on the "DNS Records" item;
- Check that the correct placeholder page - with breadcrumbs, same as in the screenshot - is now displayed.

(P.S: alternatively, you can also hack the `Dns` component to have the `showPlaceholder` prop as `true`, browsing to `/domains/manage/<site>/dns/<domain>` and changing it - React Devtools extension needed)
